src/fa/game_updater/updater.py

```
@@ -154,7 +161,7 @@ class Updater(QObject):
 
     finished = pyqtSignal()
 
-    def __init__(self, parent: QObject, worker: UpdaterWorker) -> None:
+    def __init__(self, parent: QObject, obtainer: FilesObtainer, worker: UpdaterWorker) -> None:
```

Ruff (0.14.10)

Undefined name `FilesObtainer`

(F821)